HPXMLtoOpenStudio/measure.rb

```diff
@@ -11,6 +11,7 @@
 
   require resource_file
 end
+require_relative '../ReportUtilityBills/resources/utility_bills.rb'
```

Comment:
I'd like to avoid this. Can we split this file up into a HPXMLtoOpenStudio resource file (with only the methods it needs) and a ReportUtilityBills resource file (with only the methods it needs)?